docs/proposal-004_cost-estimation.md

```diff
@@ -0,0 +1,94 @@
+> **STATUS**: This proposal is a _working draft_, it will get refined and augment as needed.
+> If any non-trivial changes are need to functionality defined here, in particular the user
+> experience, those changes should be suggested via a PR to this proposal document.
+> Any other changes to the text of the proposal or technical corrections are also very welcome.
+
```

Comment:
A few nits: augment -> augmented, need -> needed

Reply:
thanks for your help,
I copied the text from https://github.com/weaveworks/eksctl/blob/master/docs/proposal-002_existing-vpc.md
I will fix it